findbugs/src/test/java/edu/umd/cs/findbugs/DetectorsTest.java

```diff
@@ -237,9 +236,7 @@ private void setUpEngine(String... analyzeMe) throws IOException {
             String[] cpParts = classpath.split(File.pathSeparator);
             for (String cpStr : cpParts) {
                 File file = new File(cpStr);
-                if (file.isDirectory()) {
-                    project.addAuxClasspathEntry(file.getCanonicalPath());
-                }
+                project.addAuxClasspathEntry(file.getCanonicalPath());
```

Comment:
have you tested this running from the IDE? Just curious, I clearly runs from Gradle... I don't remember if it was getting this way, or `classpath` was null on that scenario